recipe/meta.yaml

```
@@ -30,10 +33,12 @@ test:
     - uuid6
   commands:
     - pip check
-    - pytest -vv test  # [not aarch64]
-    # skipping aarch64:
+    # make sure the version reported by pip is correct
+    - "pip show {{ name }} | grep -Fx 'Version: {{ version }}'" # [not win]
```

```suggestion
    - python -c 'import importlib.metadata; assert importlib.metadata.version("{name}") == "{version}"'
```
This will be cross-platform.

---

Nice thank you!